doc/api/modules.md

```diff
@@ -850,11 +850,27 @@ by the [module wrapper][]. To access it, require the `Module` module:
 const builtin = require('module').builtinModules;
 ```
 
+### module.makeRequire(filename)
+<!-- YAML
+added: REPLACEME
+-->
+
+* `filename` {string|URL} Filename for relative importing
+* Returns: {[`require`][]} Require function
+
+<!-- eslint-skip -->
+```js
+// create a require function which will resolve relative
+// to the current ES Module file
+const require = require('module').makeRequire(new URL(import.meta.url));
+```
+
```

Comment:
👆 the code example is clunky because `require` doesn't exist in ESM and you're assigning a `const` named `require` while calling the `require` function.